server/src/main/java/org/apache/cassandra/sidecar/db/RestoreJobDatabaseAccessor.java

```diff
@@ -55,13 +55,16 @@
 public class RestoreJobDatabaseAccessor extends DatabaseAccessor<RestoreJobsSchema>
 {
     private static final ObjectMapper MAPPER = new ObjectMapper();
+    public final SidecarSchema sidecarSchema;
```

Comment:
Pull this into `SidecarDatabaseAccessor` parent for the all of the classes that use `SidecarSchema`